client/state/selectors/get-billing-transaction-app-filter-values.js

```diff
@@ -0,0 +1,35 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { groupBy, map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import getBillingTransactionsByType from './get-billing-transactions-by-type';
```

Comment:
We usually import selectors by importing from the global selectors directory - `from 'state/selectors'`. Any reason to do it in this more specific way? This applies to the rest of the selectors that use other selectors.